### DISCUSSION TITLE
Provide option to publish wiki from main repo 📖

### DISCUSSION THREAD

**Discussion by tomkerkhove** [community/community#4653] [Other Feature Feedback, Questions, & Ideas]
Provide option to publish Wiki from main repo instead of a new one, for example from `docs/` or `wiki/`.

This would allow us to use the rendered wiki approach and give non-techies a UI editor while everything is consolidated in the main repo.

**Comment by PabloLION** [comment]
This also allow us to download our own wiki.
([see more feat request from me here](https://github.com/Pablion/Pablion/wiki#feature-request))

**Reply by PabloLION** [reply]
I just what I needed was always supported. There's a link under the sidebar.

**Reply by michael-hawker** [reply]
@PabloLION you can clone your own wiki repo through the link in the sidebar: https://github.com/PabloLION/PabloLION.wiki.git

That allows you to download it. You can also automate pushing to this repo as well. So you could have a workflow which grabs a directory like docs/ from your main repo and pushes that to the wiki repo if there are any changes on commits to main.

That would enable the workflow requested here in the issue as well of course.

**Reply by PabloLION** [reply]
Thanks.. I've learned that but I'm moving on from GitHub Wiki.. I've got a new list(see [here](https://note.pablolion.com/tool/note%20tool/note%20tool%20comparison%202022/#my-test-list)) to test but I didn't do it yet..
